setup.py

```diff
@@ -13,8 +13,10 @@
     long_description=open('README.rst').read(),
     url='https://github.com/mozilla/FoxPuppet',
     license='MPL2',
-    packages=['foxpuppet'],
+    packages=[
+        'foxpuppet',
+        'foxpuppet.browser'],
```

Comment:
You mean even with [find_packages](http://setuptools.readthedocs.io/en/latest/setuptools.html#using-find-packages)?